composer.json

```diff
@@ -1,47 +1,56 @@
 {
-    "name": "swentel/nostr-php",
-    "description": "Nostr helper library for PHP",
-    "homepage": "https://github.com/swentel/nostr-php",
-    "license": "GPL-2.0-or-later",
-    "authors": [
-      {
-        "name": "Sebastian Hagens",
-        "email": "info@sebastix.nl",
-        "homepage": "https://sebastix.nl",
-        "role": "Developer & maintainer"
-      },
-      {
-        "name": "Kristof De Jaeger",
-        "role": "Original author",
-        "homepage": "https://realize.be"
-      }
-    ],
-    "support": {
-      "issue": "https://github.com/swentel/nostr-php/issues",
-      "chat": "https://t.me/nostr_php"
+  "name": "swentel/nostr-php",
+  "description": "Nostr helper library for PHP",
+  "homepage": "https://github.com/swentel/nostr-php",
+  "license": "GPL-2.0-or-later",
+  "authors": [
+    {
+      "name": "Sebastian Hagens",
+      "email": "info@sebastix.nl",
+      "homepage": "https://sebastix.nl",
+      "role": "Developer & maintainer"
     },
-    "require": {
-        "php": "^8.1",
-        "ext-gmp": "*",
-        "ext-xml": "*",
-        "public-square/phpecc": "^0.1.0",
-        "bitwasp/bech32": "^0.0.1",
-        "simplito/elliptic-php": "^1.0",
-        "phrity/websocket": "^2.1.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^10",
-        "friendsofphp/php-cs-fixer": "^3.35"
-    },
-    "autoload": {
-        "psr-4": {
-            "swentel\\nostr\\": "src/"
-        }
-    },
-    "bin": [
-        "bin/nostr-php"
-    ],
-    "scripts": {
-        "lint": "@php vendor/bin/php-cs-fixer fix -v"
+    {
+      "name": "Kristof De Jaeger",
+      "role": "Original author",
+      "homepage": "https://realize.be"
+    }
+  ],
+  "support": {
+    "issue": "https://github.com/swentel/nostr-php/issues",
+    "chat": "https://t.me/nostr_php"
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/1ma/phpecc"
+    }
+  ],
+  "require": {
+    "php": "^8.1",
+    "ext-gmp": "*",
+    "ext-xml": "*",
+    "bitwasp/bech32": "^0.0.1",
+    "phrity/websocket": "^2.1.0",
+    "public-square/phpecc": "dev-fix-padding",
```

Comment:
Maybe it's better to change this `1ma/phpsecc` with a release tag until this change is merged into `public-square/phpecc` with a new release tag?

Reply:
And when I run `composer install` with these changes, there is no installable version found:

```
  Problem 1
    - Root composer.json requires public-square/phpecc dev-fix-padding, found public-square/phpecc[dev-main, dev-development, v0.1.0, v0.1.1, v0.1.2] but it does not match the constraint.
```